packages/db-mongodb/src/updateOne.ts

```diff
@@ -9,11 +9,11 @@ import { withSession } from './withSession'
 
 export const updateOne: UpdateOne = async function updateOne(
   this: MongooseAdapter,
-  { id, collection, data, locale, req = {} as PayloadRequest, where: whereArg },
+  { id, collection, data, locale, options,  req = {} as PayloadRequest, where: whereArg },
```

Comment:
I don't see where you are using `options` from the params. Did you mean to spread them in mongoseOptions on line 16?